src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,111 @@ intersection_of: part_of UBERON:0001987 ! placenta
+is_a: UBERON:0003693 ! retroperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:19:27Z xsd:dateTime
+
+[Term]
+id: UBERON:8480018
+name: posterior pararenal space
+def: "A space that is bound anteriorly by the posterior renal fascia and posteriorly by the transversalis fascia, and it only contains adipose tissue, lymphatics and blood vessels." [PMID:24065628]
+is_a: UBERON:0003693 ! retroperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:20:55Z xsd:dateTime
+
+[Term]
+id: UBERON:8480019
+name: perirectal space
+def: "An extraperitoneal space bounded laterally by the perirectal fascia, posteriorly by the posterior pelvic fascia, and anteriorly by the rectovesical fascia in the male and the rectovaginal fascia in the female." [PMID:26006061]
+is_a: UBERON:0014456 ! extraperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:22:05Z xsd:dateTime
+
+[Term]
+id: UBERON:8480020
+name: perivesical space
+def: "An extraperitoneal space circumscribed by the umbilicovesical fascia anterior and laterally, and rectovaginal (in females) or rectovesical (in males) septum posteriorly, it encounters the urachus, obliterated umbilical vessels, urinary bladder and the lower uterine segment or seminal vesicles." [PMID:24065628]
+is_a: UBERON:0014456 ! extraperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:28:46Z xsd:dateTime
+
+[Term]
+id: UBERON:8480021
+name: prevesical space
+def: "An extraperitoneal space anterior and lateral to the umbilicovesical fascia, and posterior to the trasversalis fascia. Prevesical space laterally communicates with the properitoneal space of the abdominal wall and flanks; posteroinferiorly it surrounds the lateral walls of the urinary bladder, and anteroinferiorly it forms the retropubic space (the space of Retzius)." [PMID:24065628]
+is_a: UBERON:0014456 ! extraperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:29:38Z xsd:dateTime
+
+[Term]
+id: UBERON:8480022
+name: piriform sinus
+def: "A recess on either side of the laryngeal orifice, which is bounded medially by the aryepiglottic fold, laterally by the thyroid cartilage and thyrohyoid membrane. The pyriform sinus is involved in speech." [https://en.wikipedia.org/wiki/Pyriform_sinus]
+synonym: "piriform fossa" EXACT [https://en.wikipedia.org/wiki/Pyriform_sinus]
+synonym: "piriform recess" EXACT [https://en.wikipedia.org/wiki/Pyriform_sinus]
+synonym: "pyriform fossa" EXACT [https://en.wikipedia.org/wiki/Pyriform_sinus]
+synonym: "pyriform recess" EXACT [https://en.wikipedia.org/wiki/Pyriform_sinus]
+synonym: "pyriform sinus" EXACT [https://orcid.org/0000-0002-0819-0473]
```

Comment:
a bit strange to add an orcid as dbxref for synonym, is there a paper or wiki or something that uses it?